python/ray/autoscaler/aws/example-auto-instance-type.yaml

```diff
@@ -0,0 +1,48 @@
+# EXPERIMENTAL: an example of configuring a mixed-worker cluster. Currently
+# multi-worker autoscaling only works if you use the request_resources() call.
+cluster_name: auto_instance_type
+min_workers: 1
+max_workers: 40
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+
+# Tell the autoscaler the allowed node types and the resources they provide.
+# Setting this configuration enables the Tune resource demand scheduler.
```

Comment:
Should this specify that these instance types are only used by the `request_resources` demand and not by the initial worker pool (is that accurate)? Same in the template doc.